.storybook/preview-body.html

```diff
@@ -1,6 +1,7 @@
 <style>
     #root {
         padding: 16px;
+        position: relative;
```

Comment:
Is this needed? Will customers using Preview need to add this to their stylesheets for the new styles to work?

Reply:
Adding `position: relative` here is to anchor the `position: absolute` used further down.

I'm unsure if customers will need to add this to their stylesheets.

For that reason, I've updated the code to remove it from `#root` and instead add `position: relative` to the `.be.bcpr` selector. This will leave the `preview-body.html` file unchanged.

This looks okay in Storybook (image below). However, in EUA, we add an additional header with the filename (image below).

To address this I will need to submit a PR to EUA that unsets the `be.bcpr` selector and instead adds the `position: relative` to the `.preview-container`. Seems like a better solution than adding `position: relative` to the `#root` element.

**Storybook**
![Screen Shot 2022-07-01 at 6 26 15 PM](https://user-images.githubusercontent.com/6400298/176981632-265b202f-7f0f-4d8b-8ed2-a760bdca50ab.png)

**EndUserApp would need a couple additional styles.**
![Screen Shot 2022-07-01 at 6 08 06 PM](https://user-images.githubusercontent.com/6400298/176981644-9f7b8782-c555-4226-aa18-3ec9cef3319e.png)